docs/book/src/topics/multitenancy.md

```diff
@@ -2,10 +2,16 @@
 
 To enable single controller multi-tenancy, a different Identity can be added to the Azure Cluster that will be used as the Azure Identity when creating Azure resources related to that cluster.
 
-This is achieved using the [aad-pod-identity](https://azure.github.io/aad-pod-identity) library.
+This had been achieved using the [aad-pod-identity](https://azure.github.io/aad-pod-identity) library and is dperecated now and [workload identity](https://azure.github.io/azure-workload-identity/docs/) is the next iteration of it.
+
+Note that `Service Principal With Client Password` and `Service Principal With Certificate` identitites are based on AAD pod identity which is decsribed in the below sections.
 
 ## Identity Types
 
+### Workload Identity
+
+Follow this [link](./workload-identity.md) for workload identity docs.
```

Comment:
```suggestion
Follow this [link](./workload-identity.md) for a quick start guide on setting up workload identity.

Once you've set up the management cluster with the workload identity (see link above), the corresponding values should be used to create an `AzureClusterIdentity` resource. Create an `azure-cluster-identity.yaml` file with the following content:

```yaml
apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
kind: AzureClusterIdentity
metadata:
  name: cluster-identity
spec:
  type: WorkloadIdentity
  allowedNamespaces: {}
  tenantID: <your-tenant-id>
  clientID: <your-client-id>
  allowedNamespaces: 
    list:
    - <cluster-namespace>
```

Reply:
I think re-iterating what a workload identity `AzureClusterIdentity` looks like in this doc will make sure users don't miss it and go directly to the other example yamls below